src/tests/ftest/io/nvme_io.py

```diff
@@ -51,6 +52,7 @@ class NvmeIo(avocado.Test):
     def setUp(self):
```

Comment:
I am also working on changing this as per of PR#400 (Code updated with apricot class), so we might have to rebase based on what ever patch land first